airgun/entities/report_template.py

```diff
@@ -55,6 +56,27 @@ def unlock(self, entity_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def export(self, entity_name):
+        """Export report template.
+
+         :return str: path to saved file
+        """
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Actions'].widget.fill('Export')
+        view.flash.assert_no_error()
```

Comment:
that is redundant as it just can't happen

Reply:
How do you know? I'm not saying that you're wrong but how can you know some error won't happen, let alone in the future releases?

Reply:
Once you do `view.table.row(name=entity_name)['Actions'].widget.fill('Export')` you lose control on a browser completely as opened dialog block Selenium from access till it closed, so you can't read anything no matter what happen

Reply:
That makes sense, I'll remove the line.